### DISCUSSION TITLE
Create Age Gate for Erotic Games

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#743] [Ideas]
There are a number of erotic games for various system, both licenced and some of them contain nudity.

While I don't think anyone has argued against removing or prohibiting work on them it would be appropriate to make an age gate specifically for these games.

Luchaos pointed out that it would be better for such sets to be hidden by default and then be opt-in. This seems like the best way to handle it.

I also think it would be easier to handle this site wise, but not sure about RAemulators and Retroarch wise.

**Comment by CoolCanuck** [comment]
How far does this reach? 
Does it prohibit showing the game name and icon on the feed for users who have not opted in? Does it prohibit seeing friends RP playing an erotic game?

**Comment by ghost** [comment]
I suggest a option in everyones Profile to hide Erotic content on the Site, this includes:

- Game List
- Active Players (Rich Presence)
- Global Feed
- Leaderboards
- Search results

Some of us Retro Gamers have childrens they play retro games with and the last thing parents need is erotic content being displayed anywhere on the Site.

Erotic content should also be hidden for unregistered users.

**Comment by kdecks** [comment]
Even if some erotic games would be hidden from users automatically, I would push for retaining our our "PG" policy on achievement images/text for the sole reason that I don't want for me other others to be submitted to the pornography or sexuality of other users on our platform.

If someone wants to play an erotic game that's fine by me.

**Comment by Jamiras** [comment]
A mature content warning was added by #904. It currently only blocks access to the game page itself. It needs to be expanded to achievements belonging to the game, game lists, and possibly links to the games.
